### PR TITLE
Added support for videos

### PR DESCRIPTION
**Pull request by harrisonmiller**
Hello!

I added support for videos to the extension.  Supported file types are mp4, webm, and ogg.  I create a HTML video tag using the provided url as the source and then use [requestAnimationFrame](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame) to draw the video frame by frame.  The default attributes are muted, autoplay, and loop.  Let me know what you think.  I figured it might be useful to some others using the extension if it is to my team. 

You can see an example in action [here](https://knowledgeforum.org/html/demoview.html).  They are currently locked but resizing and moving them around works the same as the images when they are unlocked. 

Harrison

### CODE REVIEW

File: cytoscape-supportimages.js [hunk 1]
```diff
@@ -693,18 +710,32 @@
...
             supportImage.bounds.width = supportImage.bounds.width || w;
             supportImage.bounds.height = supportImage.bounds.height || h;
+
+            if(img.readyState >= 0){
+              img.addEventListener('canplaythrough', function(){
+                img.play();
+              });
+
+              function animateFrames(){
+                context.drawImage(img, supportImage.bounds.x, supportImage.bounds.y, supportImage.bounds.width, supportImage.bounds.height);
```
Comment (jhonatandarosa) [inline cytoscape-supportimages.js]: Needs to include the part that draws the controls for resize too.
When a video is unlocked and selected the resize controls stays underneath the video frame.

**Review by jhonatandarosa — CHANGES_REQUESTED** [review]
Besides that one point of the resize controls underneath the video frame, LGTM. Nice work!

File: cytoscape-supportimages.js (hunk 1 above)
Comment (harrisonmiller) [reply cytoscape-supportimages.js]: Good catch, it is fixed.  Thank you!

**Review by jhonatandarosa — APPROVED** [review]

**Comment by jhonatandarosa** [comment]
Just published the version 1.0.2 with this feature.
Thanks

**Pull request merged**